.circleci/config.yml

```diff
@@ -1298,7 +1298,7 @@ workflows:
           xcode_version: '14.3.0'
           <<: *release-tags
       - deploy-purchase-tester:
-          xcode_version: '14.3.0'
+          xcode_version: '15.2'
```

Comment:
I wonder if we should update all the others (except the tests in older versions of course)... I think ideally we default to this one and just overwrite it where needed... But this looks good for now!

Reply:
yes, I think we should. This is the one that failed when deploying so fixing this for now

Reply:
https://github.com/RevenueCat/purchases-ios/pull/3870